examples/bank/bank.go

```diff
@@ -182,34 +189,55 @@ func (bank *Bank) initBankAccounts(cash int64) {
 			return err
 		}
 		for i := 0; i < bank.numAccounts; i++ {
-			batch.Put(bank.makeAccountID(i), value)
+			id := bank.makeAccountID(i)
+			if _, ok := accts[string(id)]; !ok {
+				batch.Put(id, value)
+				initCash += cash
+			}
 		}
 		return tx.Run(batch)
 	}); err != nil {
 		log.Fatal(err)
 	}
 	log.Info("done initializing all accounts")
+	return initCash
 }
 
 func (bank *Bank) periodicallyCheckBalances(initCash int64) {
+	var lastNumTransfers int32
+	lastNow := time.Now()
 	for {
 		// Sleep for a bit to allow money transfers to happen in the background.
 		time.Sleep(time.Second)
-		fmt.Printf("%d transfers were executed.\n\n", bank.numTransfers)
+		now := time.Now()
+		elapsed := now.Sub(lastNow)
+		numTransfers := atomic.LoadInt32(&bank.numTransfers)
+		fmt.Printf("%d transfers were executed at %.1f/second.\n", bank.numTransfers,
```

Comment:
fmt.Printf("Last %d transfers .......", numTransfers-lastNumTransfers,

will print the number of transactions in the last epoch which is probably better.